src/components/Profile/components/SchedulePanel/index.tsx

```diff
@@ -48,11 +49,27 @@ const GordonSchedulePanel = ({ profile, myProf }: Props) => {
     });
   }, [profile.AD_Username]);
 
+  useEffect(() => {
+    const getAccordionState = localStorage.getItem('accordionState');
+    if (getAccordionState !== null) {
+      setAccordionState(getAccordionState === 'true');
+    }
+  }, []);
+
+  const toggleAccordionState = () => {
+    setAccordionState(!accordionState);
+    localStorage.setItem('accordionState', String(!accordionState));
```

Comment:
For keys like this that need to be the same everywhere they're referenced, the best practice is to make it a constant value outside of the component, e.g. on line 25:
```typescript
    const scheduleOpenKey = 'profile.schedule.isOpen';
```
Since the key needs to be unique across all of 360 for all time, I made it a little bit more specific as well.

Reply:
I am a little confused on what this is suggesting and i'm hoping you can clarify for me, otherwise I have made changes based on the other comments.